docs/docs/concepts/infrastructure-architecture.mdx

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 1
+---
+
+# Infrastructure Architecture
+
+JHub Apps, also known as JupyterHub Apps Launcher, is an [Externally-Managed JupyterHub service](https://jupyterhub.readthedocs.io/en/stable/reference/services.html#externally-managed-services) designed to enable users to launch various server types including API services, dashboards such as Panel, Bokeh, Streamlit, standard JupyterLab instances, and any generic Python command.
```

Comment:
```suggestion
JHub Apps, also known as JupyterHub (Jhub) App Launcher, is an [Externally-Managed JupyterHub service](https://jupyterhub.readthedocs.io/en/stable/reference/services.html#externally-managed-services) designed to enable users to launch various server types including API services, any generic Python command, standard JupyterLab instances, and dashboards such as Panel, Bokeh, Streamlit.
```

I reorganized the order of the server types so that it would read a little more smoothly. I've also added the abbreviation for JupyterHub, though I'm not convinced that needs to be done in this exact spot, it does need to be done early on though.